blockchain/reactor.go

```diff
@@ -311,6 +325,14 @@ func (m *bcBlockRequestMessage) String() string {
 	return cmn.Fmt("[bcBlockRequestMessage %v]", m.Height)
 }
 
+type bcNoBlockResponseMessage struct {
```

Comment:
Needs to be registered with go-wire. See call to `RegisterInterface`

Reply:
Roger that, updating, thanks.

Reply:
Added, thank you!